src/Controller/FightController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\AttackManager;
+use App\Model\UnicornManager;
+
+class FightController extends AbstractController
+{
+    /**
+     * Display about page
+     */
+    public function index(): string
+    {
+        $unicornManager = new AttackManager();
```

Comment:
coherence between variables and Object, please change unicornManager for attackManager